### PR TITLE
Add bundle objects

### PR DESCRIPTION
**Pull request by Justus-Saringer**

### CODE REVIEW

File: code/states/BundleObjects.kt [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        text = "My name is " + person.name + " " + person.lastname,
+        modifier = Modifier
+            .width(200.dp)
+            .background(Purple200),
+        textAlign = TextAlign.Center
+    )
+}
+
+@Composable
+private fun RiverText(river: MutableState<River>) {
```
Comment (Tautellini) [inline code/states/BundleObjects.kt:61]: You dont modify the state of river here. So please dont pass the state itself. Also if you would need to, please favor a lambda wherever possible. Also this is a suboptimal naming right? You dont know at this point you are talking about a ListSaver.

**Review by Tautellini — CHANGES_REQUESTED** [review]
see comments

File: code/states/BundleObjects.kt (hunk 1 above)
Comment (Justus-Saringer) [reply code/states/BundleObjects.kt]: I adopted the comment. I know the naming is not on top of the world, but I don't know how to do it better. I tried to improve it.

File: demo/src/main/java/de/check24/compose/demo/features/bundle/AndroidUIBundleObjectsActivity.kt [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = BundleObjectsBinding.inflate(layoutInflater)
+        setContentView(binding?.root)
+        supportActionBar?.title = "Bundle Objects"
+
+        binding?.textview1?.text = person
+        binding?.textview2?.text = river1
+        binding?.textView3?.text = river2
```
Comment (Darius3001) [inline demo/src/main/java/de/check24/compose/demo/features/bundle/AndroidUIBundleObjectsActivity.kt:22]: still a fan of apply 😄

File: demo/src/main/java/de/check24/compose/demo/features/bundle/AndroidUIBundleObjectsActivity.kt [hunk 3]
```diff
@@ -0,0 +1,24 @@
+package de.check24.compose.demo.features.bundle
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import de.check24.compose.demo.databinding.BundleObjectsBinding
+
+class AndroidUIBundleObjectsActivity : AppCompatActivity() {
+
```
Comment (Darius3001) [inline demo/src/main/java/de/check24/compose/demo/features/bundle/AndroidUIBundleObjectsActivity.kt:8]: you dont really use bundle. We should rethink that and also maybe another example.

**Review by Darius3001 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**